Automattic/wp-calypso
# Command Palette Exploration: Add the command palette to WPCOM SMP

WORK IN PROGRESS

- Related to https://github.com/Automattic/dotcom-forge/issues/3964

## Proposed Changes

* Add Gutenberg Command Palette to SMP calypso

## Testing Instructions



* Navigate to `/sites`
* Press `cmd + k` on Mac or `ctrl + k` on Windows
* Observe the new Command Palette.

## Screencast


https://github.com/Automattic/wp-calypso/assets/779993/7e0394fa-3ed1-4bfa-bce4-8fbfb021f350






## Pre-merge Checklist
